frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -122,7 +122,7 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
 
         const source = utils.getOptimizedContent(selectedEntity.machinery_original, selectedEntity.format);
 
-        if (source !== this.props.terms.sourceString) {
+        if (source !== this.props.terms.sourceString && parameters.project !== 'terminology') {
```

Comment:
Is the project name `terminology` a mozilla-ism of Pontoon, or is that hard-coded, and other instances can't have a non-terminology project anyway?

Reply:
It's hardcoded for all locales. Same as the "Tutorial" project.

Reply:
Maybe there should be a "reserved words" list somewhere in the documentation to make this explicit to external users?

Reply:
Obviously, the Admin will complain if you add a project which already exists, but I agree it's helpful to document that.

Maybe we can list internal projects here?
https://mozilla-pontoon.readthedocs.io/en/latest/user/localizing-your-projects.html#adding-a-new-project-to-pontoon